cirq/google/sim/xmon_simulator.py

```diff
@@ -126,7 +127,7 @@ def _run(
         param_resolver: study.ParamResolver,
         repetitions: int,
     ) -> Dict[str, List[np.ndarray]]:
-        """Runs the entire supplied Circuit, mimicking the quantum hardware."""
+        """See definition in `sim.SimulatesSamples`."""
```

Comment:
Prefer `cirq.SimulatesSamples`, because that is how the user will think about it.